paymentsheet/src/main/java/com/stripe/android/paymentsheet/paymentdatacollection/cvcrecollection/CvcRecollectionViewModel.kt

```diff
@@ -50,6 +53,16 @@ internal class CvcRecollectionViewModel(args: Args) : ViewModel() {
         }
     }
 
+    private fun onCvcChanged(cvc: String) {
+        viewModelScope.launch {
```

Comment:
Why does this need to be in a coroutine?

Reply:
It doesn't. I just pushed a fix for it